app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -124,8 +125,8 @@ public DocumentEditFragment() {
     public void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         final Bundle args = getArguments();
-        if (_savedInstanceState != null && _savedInstanceState.containsKey(SAVESTATE_DOCUMENT)) {
-            _document = (Document) _savedInstanceState.getSerializable(SAVESTATE_DOCUMENT);
+        if (savedInstanceState != null && savedInstanceState.containsKey(SAVESTATE_DOCUMENT)) {
+            _document = (Document) savedInstanceState.getSerializable(SAVESTATE_DOCUMENT);
```

Comment:
While developing #1809 I noticed that there is an issue with rotation at the viewmode vs editmode state.  
So currently on master when you **enter** as editor, go to viewmode, rotate the device, you are in the editor again.  
Also the opposite when you  **enter** in viewmode, rotating the device will always go back to viewmode.

Guess also needs some savedInstance code?